v3/mqtt_dummy.go

```diff
@@ -212,29 +237,38 @@ func sendPublish(client *dummyClient, pubCmd DummyCmd) {
 				Payload: globalConf.SubData,
 				QOS:     packet.QOSAtMostOnce,
 			}
-			pubPkt.PacketID = 1
+			pubPkt.PacketID = uint16(atomic.AddUint32(&packetID, 1))
 			pkts = append(pkts, pubPkt)
 		}
-	case PublishKvCmd:
+	case PublishKvSync:
```

Comment:
Instead of one command to publish a KV Sync, we now have KVSync, KVSet, and KVDelete.